.github/workflows/integration-tests.yml

```diff
@@ -82,44 +82,44 @@ jobs:
 #      - name: Run test
 #        run: bazel run //e2e:BazelBspActionGraphV2Test
 #
-#  java-8-project-test:
-#    runs-on: ubuntu-latest
-#    needs: setup
-#    steps:
-#      - uses: actions/checkout@v2
-#      - name: Set up JDK 11
-#        uses: actions/setup-java@v1
-#        with:
-#          java-version: 11
-#      - name: Cache bazel
-#        uses: actions/cache@v2
-#        env:
-#          cache-name: cache-bazel
-#        with:
-#          path: ~/.cache/bazel
-#          key: ${{ runner.os }}-${{ env.cache-name }}-${{ github.sha }}
-#      - name: Run test
-#        run: bazel run //e2e:BazelBspJava8ProjectTest
-#
-#  java-11-project-test:
-#    runs-on: ubuntu-latest
-#    needs: setup
-#    steps:
-#      - uses: actions/checkout@v2
-#      - name: Set up JDK 11
-#        uses: actions/setup-java@v1
-#        with:
-#          java-version: 11
-#      - name: Cache bazel
-#        uses: actions/cache@v2
-#        env:
-#          cache-name: cache-bazel
-#        with:
-#          path: ~/.cache/bazel
-#          key: ${{ runner.os }}-${{ env.cache-name }}-${{ github.sha }}
-#      - name: Run test
-#        run: bazel run //e2e:BazelBspJava11ProjectTest
-#
+  local-jdk-test:
+    runs-on: ubuntu-latest
+    needs: setup
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 17
+      - name: Cache bazel
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-bazel
+        with:
+          path: ~/.cache/bazel
+          key: ${{ runner.os }}-${{ env.cache-name }}-${{ github.sha }}
+      - name: Run test
+        run: bazel run //e2e:LocalJdkTest
+
+  remote-jdk-test:
+    runs-on: ubuntu-latest
+    needs: setup
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```

Comment:
```suggestion
          java-version: 8
```

Reply:
it is done I guess it is not updated in git

Reply:
are u sure?